fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/jobs/setloandelinquencytags/SetLoanDelinquencyTagsTasklet.java

```diff
@@ -36,16 +36,19 @@
 @RequiredArgsConstructor
 public class SetLoanDelinquencyTagsTasklet implements Tasklet {
 
-    private final LoanProductRepository loanProductRepository;
     private final DelinquencyWritePlatformService delinquencyWritePlatformService;
     private final LoanReadPlatformService loanReadPlatformService;
 
     @Override
     public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
+        ThreadLocalContextUtil.setActionContext(ActionContext.COB);
+        log.info("Run job for date {}", DateUtils.getBusinessLocalDate());
         Collection<LoanScheduleDelinquencyData> loanScheduleDelinquencyData = this.loanReadPlatformService
                 .retrieveScheduleDelinquencyData(DateUtils.getBusinessLocalDate());
-        log.debug("Were found {} items", loanScheduleDelinquencyData.size());
+        log.info("Were found {} items", loanScheduleDelinquencyData.size());
```

Comment:
This should be debug level logging (IMO)

Reply:
Done